open-apparel-registry/open-apparel-registry
# Add sector to facility claim

## Overview

Adds the ability to set sector on approved facility claims, and uses facility claims sectors when updating facility indexes.

Connects #1911 

## Demo

![submit-next-steps](https://user-images.githubusercontent.com/4432106/175374689-d454367b-d5a8-4d77-bd20-02477a926d57.gif)


## Notes

I also added one piece of functionality not in the issue - I updated the sector choice API from #1871 to include sectors from approved facility claims.

## Testing Instructions

* `scripts/server`
* As `c3@example.com` claim a facility
* As `c1@example.com` approve the claim (I used an Incognito window)
* As `c3@example.com` update your claim to include a sector other than "Apparel". You should be able to both use the autocomplete to choose from existing sectors as well as add new ones.
* On the dashboard, search for the sector you just entered. You should see the facility you claimed.


## Checklist
